moksha.hub/moksha/hub/stomp/protocol.py

```diff
@@ -69,10 +69,12 @@ def connected(self, msg):
 
     def subscribe(self, dest, **headers):
         f = stomper.Frame()
+        # https://stomp.github.io/stomp-specification-1.2.html#SUBSCRIBE_ack_Header
+        ack = self.client.hub.config.get('stomp_ack_mode', 'auto')
```

Comment:
Would we want to check that it is one of "auto", "client" or "client-individual" or just leave it for stomper error handling? Not entirely sure how the error message will look like here.